src/pages/Settings/index.js

```diff
@@ -119,100 +120,136 @@ const NotificationSettings = (
-                      checked={showTreeToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-                <tr>
-                  <td>Check connectivity</td>
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showConnectionStatusToast"
-                      id="showConnectionStatusToast-input"
-                      checked={showConnectionStatusToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-                <tr>
-                  <td>Use CTRL and scroll to zoom</td>
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showCtrlToZoomToast"
-                      id="showCtrlToZoomToast-input"
-                      checked={showCtrlToZoomToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-                <tr>
-                  <td>Customise Settings</td>
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showSettingsToast"
-                      id="showSettingsToast-input"
-                      checked={showSettingsToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-                <tr>
-                  <td>Help Links</td>
-                  <td>
-                    <input
-                      type="checkbox"
-                      name="showHelpToast"
-                      id="showHelpToast-input"
-                      checked={showHelpToast}
-                      onChange={noop}
-                    />
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+) => {
+  const [status /*: string */, setStatus] = useState(
+    Notification.permission === 'granted' ||
+      Notification.permission === 'denied'
+      ? 'answered'
+      : 'not yet',
+  );
+  return (
+    <form data-category="notifications">
+      <h4>Notification settings</h4>
+      <SchemaOrgData
+        data={{
+          name: 'Notification settings',
+          description: 'Notifications in the website can be turned on/off.',
+        }}
+        processData={schemaProcessDataPageSection}
+      />
+      <div className={f('row')}>
+        <div className={f('medium-12', 'column')}>
+          {status === 'answered' ? (
+            <>
+              <p>
+                To change notification preference, please do in the browser
+                settings.
+              </p>
+              <button className={f('button')} disabled>
+                Notifications {Notification.permission}
+              </button>
+            </>
+          ) : (
+            <>
+              <p>
+                Allow browser to notify when the InterProScan search has
```

Comment:
Not sure of the wording here. Maybe something like:

Allow us to send you browser notifications when one of your Jobs or Downloads finishes.

@CodeGit what do you think?